src/web-worker/amp-worker.js

```diff
@@ -103,7 +103,19 @@ class AmpWorker {
           type: 'text/javascript',
         });
         const blobUrl = win.URL.createObjectURL(blob);
-        this.worker_ = new win.Worker(blobUrl);
+        if (self.trustedTypes && self.trustedTypes.createPolicy) {
```

Comment:
Can we add a comment here about why we trust this input?

Reply:
Added "We can trust the url for this policy usage because the blobUrl pulls the script from a controlled source, the ww.js file."